datasets/ONS-International-trade-in-services-by-subnational-areas-of-the-UK/codelists/country-or-origin-of-trade.csv

```diff
@@ -0,0 +1,4 @@
+Label,Notation,Parent Notation,Sort Priority,Description
+Total,total,,1,
+EU,eu,total,2,"The European Union consists of 28 member countries including the United Kingdom. For trade purposes, this includes all 27 countries other than the UK as well as the European Central Bank and European Institutions."
```

Comment:
I'm a bit concerned that these values will have to be changed in a couple of years time and we have no way to update this.